rfcs/2021-05-17-7469-load-balancing-k8s-aggregators.md

```diff
@@ -0,0 +1,110 @@
+# RFC 7469 - 2021-05-16 - Scaling and load balancing for Vector aggregators in Kubernetes
+
+This RFC describes the need for a tooling and environment agnostic load balancing solution to be bundled with the Vector aggregator Helm chart.
+
+* [Scope](#scope)
+* [Out of Scope](#out-of-scope)
+* [Motivation](#motivation)
+* [Internal Proposal](#internal-proposal)
+* [Rationale](#rationale)
+* [Prior Art](#prior-art)
+* [Drawbacks](#drawbacks)
+* [Alternatives](#alternatives)
+  * [Do Nothing](#do-nothing)
+  * [Only Client-side Load Balancing](#only-client-side-load-balancing)
+  * [Require a Service Mesh](#require-a-service-mesh)
+* [Outstanding Questions](#outstanding-questions)
+* [Plan Of Attack](#plan-of-attack)
+
+## Scope
+
+Load balancing will be a concern for any `sink` or `source` supported by Vector; some of which can use a general solution (ex: load balancing for our HTTP based `sinks`) and some of which are specific to the component (ex: Kafka `source`/`sink`). Due to the breadth of the topic, this RFC will focus on three specific cases for load balancing to Vector aggregators in Kubernetes, while also giving consideration to future adoption for other components.
+
+* Vector agent to Vector aggregator
+* Datadog agent to Vector aggregator
+* Syslog agent to Vector aggregator
```

Comment:
Syslog is mentioned here, and once below, but UDP sources other times.  It that really referring to the same thing?

You might want to call out syslog UDP vs syslog TCP.

Reply:
Correct - Syslog TCP only, I'll make a quick update on that line